helm/prometheus-rules/templates/alerting-rules/mimir.rules.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
```

Comment:
I guess this is fine for now but I'd really like if we could move to slos instead

Reply:
So if a component is down, that would be a working hour alert but if the mimir service is not working, we get paged during the night?